src/main/sentry.ts

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable @typescript-eslint/no-require-imports */
+export function initSentry(): void {
```

Comment:
you should probably put it outside of main as it's used in both?

Reply:
I think it should be in both